scripts/import.ts

```diff
@@ -156,15 +335,16 @@ const validateIcon = async (path: string, iconName: string): Promise<IconValidit
       throw new Error(`Expect icon dimensions to be 18x18, found ${width}x${height}`);
     }
     
-    // TODO: accessibility checks? E.g. each icon should have an accessible name.
```

Comment:
If we no longer check for icon name, we don't have to accept `iconName` as the second parameter in `validateIcon` function?

Reply:
Might be useful in the future. But I've removed it for now, can always add it back if needed.